src/mui/input/CheckboxGroupInput.js

```diff
@@ -60,38 +77,39 @@ export class CheckboxGroupInput extends Component {
         }
     };
 
-    render() {
+    renderCheckbox = (choice) => {
```

Comment:
As it's not used outside of `<CheckboxGoupInput>`, and it's tested there, I think it's a hassle that we can avoid. Same for the two others.